### PR TITLE
New README format

### PR DESCRIPTION
**Pull request by autero1**
This PR updates the READMEs to the new format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,45 @@
+<!--
+:type: service
+:name: Google Cloud Load Balancer
+:description: Deploy Google's Cloud Load Balancer, with support for HTTP/HTTPS, internal and network load balancers.
+:icon: /_docs/cloud-load-balancer-icon.png
+:category: loadbalancer
```
Comment (yorinasub17) [inline README.md]:
```suggestion
:category: load-balancing
```

The pattern we are using is to kebob case the category from the spreadsheet.

File: modules/http-load-balancer/README.md [hunk 2]
```diff
@@ -1,78 +1,75 @@
 # HTTP(S) Load Balancer Module
```
Comment (yorinasub17) [inline modules/http-load-balancer/README.md]: I think we only needed to update the README of the root repo, unless there was a service in the catalog that mapped to an individual module. With that said, this is a much nicer format overall so still good to update these as well! The only downside is the duplicated boilerplate text...

Comment (autero1) [reply modules/http-load-balancer/README.md]: Ah, ok. Think I misunderstood that part. Why I'm tempted to revert to the old format is that the original module documentation contains module usage instructions and already had a lot of the core concepts embedded. What do you think?

Comment (yorinasub17) [reply modules/http-load-balancer/README.md]: Ah that's a good point. That also probably looks better in the terraform registry too. I am in agreement with you about reverting.

**Comment by robmorgan** [comment]
Also LGTM!

**Comment by autero1** [comment]
Thanks for the review, merging this in!

**Pull request merged**